src/shared/helpers/status-helpers.ts

```diff
@@ -0,0 +1,3 @@
+import { ErrorContent } from '../layouts/error';
+
+export const hasClientError = (error: ErrorContent | undefined) => error && (error.status >= 400 && error.status < 500);
```

Comment:
To make it more strict?
```suggestion
export const hasClientError: boolean = (error: ErrorContent | undefined) => error && (error.status >= 400 && error.status < 500);
```